src/Microsoft.AspNetCore.SystemWebAdapters/AppConfiguration.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Web;
+using Microsoft.AspNetCore.SystemWebAdapters.Adapters;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters;
+
+#if !NET45
+/// <summary>
+/// Provides a static way to access application configuration settings while migrating to Microsoft.Extensions.Configuration types.
+/// </summary>
+public static class AppConfiguration
```

Comment:
Do we want to mark this as experimental or are we happy with its shape?

Reply:
it's a simple enough shape that don't expect to change drastically or in a breaking way